assets/js/screens/Collection/CollectionImageModal.jsx

```diff
@@ -0,0 +1,136 @@
+  const [selectedWork, setSelectedWork] = useState();
+  const [filteredWorkImages, setFilteredWorkImages] = useState();
+  useEffect(() => {
+    setFilteredWorkImages(collection ? collection.works : []);
+  }, []);
+
+  const handleFilterChange = (e) => {
+    const filterValue = e.target.value.toUpperCase();
+    if (!filterValue) {
+      return setFilteredWorkImages(collection ? collection.works : []);
+    }
+    const filteredList = collection.works.filter((work) => {
+      return work.descriptiveMetadata.title
+        ? work.descriptiveMetadata.title.toUpperCase().indexOf(filterValue) > -1
+        : false;
+    });
+    setFilteredWorkImages(filteredList);
+  };
+
+  let [
+    setCollectionImage,
+    { loading, error: mutationError, data },
+  ] = useMutation(SET_COLLECTION_IMAGE, {
+    onCompleted({ setCollectionImage }) {
+      toastWrapper("is-success", "Collection image has been updated");
+      handleClose();
+    },
+    onError(error) {
+      console.log("Error: ", error);
+    },
+  });
+
+  const setWork = (workId) => {
+    setSelectedWork(workId);
+    return false;
+  };
+
+  const handleThumbnailChange = () => {
+    if (!selectedWork) {
+      toastWrapper("is-danger", `Please select an image to add to collection`);
+      return false;
+    }
+    setCollectionImage({
+      variables: { collectionId: collection.id, workId: selectedWork },
+    });
+  };
+
+  const styles = {
+    highlightImage: { outline: "10px solid #4e2a84 " },
+    fullWidth: { width: "85%" },
+  };
+
+  return (
+    <div
+      className={`modal ${isModalOpen ? "is-active" : ""}`}
+      data-testid="modal-collection-thumbnail"
+    >
+      <div className="modal-background"></div>
+      <div className="modal-card" style={styles.fullWidth}>
+        <header className="modal-card-head">
+          <p className="modal-card-title">
+            Representative Image for Collection
+          </p>
+          <button
+            className="modal-close is-large"
+            aria-label="close"
+            onClick={handleClose}
+          ></button>
+        </header>
+        <section className="modal-card-body">
+          <div className="control">
+            <input
+              className="input"
+              onChange={handleFilterChange}
+              placeholder="Filter collections"
+              data-testid="input-collection-filter"
+            />
+          </div>
+          <div className="section columns is-multiline">
+            {filteredWorkImages &&
+              filteredWorkImages.map((work) => (
+                <div
+                  key={work.id}
+                  className="column  is-narrow"
+                  style={selectedWork == work.id ? styles.highlightImage : {}}
+                  onClick={() => {
+                    setWork(work.id);
+                  }}
+                >
+                  <figure style={{ width: "128px", height: "128px" }}>
```

Comment:
Instead of an inline `style`, let's use Bulma's helper variables: https://bulma.io/documentation/elements/image/#fixed-square-images